moveit_ros/benchmarks/scripts/moveit_benchmark_statistics.py

```diff
@@ -130,6 +130,22 @@ def readBenchmarkLog(dbname, filenames):
         (runid INTEGER, time REAL, PRIMARY KEY (runid, time),
         FOREIGN KEY (runid) REFERENCES runs(id) ON DELETE CASCADE)""")
 
+    allExperimentsName = "all_experiments"
+    allTotalTime = 0.0
+    timelimit = 0.0
+    memorylimit = 0.0
+    allNrruns = 0
```

Comment:
what does allNrruns stand for? 

these variables are not in ROS / MoveIt style...